cargo-concordium/src/main.rs

```diff
@@ -83,6 +84,54 @@ enum Command {
         )]
         path: PathBuf,
     },
+    #[structopt(
+        name = "schema-json",
+        about = "Convert a schema into its JSON representation and output it to a file.
+        A schema has to be provided either as part of a smart contract module or with the schema \
+                 flag. You need to use exactly one of the two flags(`--schema` or `--module`) \
+                 with this command."
+    )]
+    SchemaJSON {
+        #[structopt(
+            name = "out",
+            long = "out",
+            short = "o",
+            default_value = ".",
+            help = "Writes the converted JSON representation of the schema to files named after \
+                    the smart contract names at the specified location. Directory path must \
+                    exist. (expected input: `./my/path/`)."
+        )]
+        out:          PathBuf,
+        #[structopt(
+            name = "schema",
+            long = "schema",
+            short = "s",
+            conflicts_with = "module",
+            required_unless = "module",
+            help = "Path and filename to a file with a schema (expected input: \
+                    `./my/path/schema.bin`)."
+        )]
+        schema_path:  Option<PathBuf>,
+        #[structopt(
+            name = "wasm-version",
+            long = "wasm-version",
+            conflicts_with = "module",
```

Comment:
I removed this alltogether and now use it both when supplying the schema, and the module. In case somebody supplies the old module without an explicit version.